rasa/core/policies/ted_policy.py

```diff
@@ -191,7 +191,7 @@ class TEDPolicy(Policy):
         # Max position for relative embeddings
         MAX_RELATIVE_POSITION: None,
         # Use a unidirectional or bidirectional encoder.
-        UNIDIRECTIONAL_ENCODER: True,
+        UNIDIRECTIONAL_ENCODER: False,
```

Comment:
Will need a change in the docs too. Can you also mention the change of default value in the changelog?

Reply:
this parameter was never in the docs, because the fact that it was configurable is rather a bug

Reply:
created bug fix changelog